app/assets/stylesheets/helpers/_core-print.scss

```diff
@@ -7,14 +7,14 @@ main > header {
   position: relative;
 
   h1 {
-    @include bold-24;
+    @include govuk-font(24, $weight: bold);
```

Comment:
Hi @andysellick as discussed, while your suggestions are sensible as always, team consensus is that we should not introduce these types of changes as part of this piece of work. 

Firstly, because it does not fit the scope of this PR.
Secondly, because although removing/refactoring those styles seems like an easy task on the surface,  there is potentially a large amount of work in ensuring we don't break anything in the process (searching all the applications for the thing, removing the dependency for the thing before removing it from `static` – implementing this can be time consuming if _the thing_ is actually many classes/selectors).

Following on from this I've raised [an issue](https://github.com/alphagov/static/issues/2239) to document the points you've raised.